open-craft/sprints
# [BB-1245] Retrieve vacation time from Google Calendar

This adds auto-setup of vacations defined in Google Calendar.
**Note**: Currently this PR's target branch is `agrendalath/bb-1244-sprint-dashboard` for allowing parallel reviews. I'll switch this to `master` after merging that branch.

## Testing instructions:
1. Set `JIRA_SERVER`, `JIRA_USERNAME` and `JIRA_PASSWORD` in your envs (e.g. in `.envs/.local/.django`).
1. Set `GOOGLE_API_PRIVATE_KEY` and `GOOGLE_API_CLIENT_EMAIL` (I provided them in the ticket) in your envs (e.g. in `.envs/.local/.django`).
1. Start the backend with `docker-compose -f local.yml up --build`.
1. Navigate to `sprints/frontend` and run `npm i && npm start`. Check the `Vacation` column in the dashboard.
